src/Microsoft.ML.Data/DataLoadSave/Binary/BinaryLoader.cs

```diff
@@ -959,18 +960,15 @@ void ICanSaveModel.Save(ModelSaveContext ctx)
         /// Write the parameters of a loader to the save context. Can be called by <see cref="SaveInstance"/>, where there's no actual
         /// loader, only default parameters.
         /// </summary>
-        private static void SaveParameters(ModelSaveContext ctx, int threads, string generatedRowIndexName, Double shuffleBlocks)
+        private static void SaveParameters(ModelSaveContext ctx, int threads, Double shuffleBlocks)
         {
             // *** Binary format **
             // int: Number of threads if explicitly defined, or 0 if the
             //      number of threads was automatically determined
-            // int: Id of the generated row index name (can be null)
             // Double: The randomness coefficient.
```

Comment:
>Double: The randomness coefficient. [](start = 15, length = 35)

I noticed that this line doesn't appear in the load method, can you please add it?